postgres/datadog_checks/postgres/metadata.py

```diff
@@ -2,7 +2,8 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import time
-from typing import Dict, List, Optional, Tuple, Union  # noqa: F401
+from typing import Dict, List, Optional, Tuple, Union
+from datadog_checks.postgres.connections import MultiDatabaseConnectionPool  # noqa: F401
```

Comment:
> # noqa: F401

comment looks like it ended up on wrong line here, but also im not sure what this comment refers to anyways